### ISSUE TITLE
Accessibility - My Profile Screen - TalkBack Not reading Reuired Fields

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/OCIO-DOLSafety-Android#34]
TalkBack is not reading the asterisks for the required fields at all when they should be announced as “Required”.

**Comment by greg-gruse** [comment]
This was fixed in the build dated July 24th see info screen for date

**Comment by jfc3-dol** [comment]
Thanks, I will check it next time I review the application which should be later in the week.
